python/tvm/dlight/gpu/matmul.py

```diff
@@ -328,7 +328,7 @@ def is_spatial(block: BlockRV) -> bool:
         by, vy, ty, yi = sch.split(y, [None, vthread_y, block_size_y, micro_size_y])
         ko, ki = sch.split(k, factors=[None, micro_size_k])
         sch.reorder(bx, by, vy, vx, ty, tx, ko, ki, yi, xi)
-        sch.bind(batch, "blockIdx.z")
+        bx = sch.fuse(batch, bx)
```

Comment:
I suppose fusing `batch` with `by` will more faithfully inherit the semantics of `sch.bind(batch, "blockIdx.z")` because the physical id is `blockIdx.z * (gridDim.x * gridDim.y) + blockIdx.y * (gridDim.x) + blockIdx.x`.

Reply:
Yes you're right. Just updated. Does it look good to you?

Reply:
It looks great, thank you!